src/melee/mn/mn_2295.h

```diff
@@ -3,6 +3,37 @@
 
 #include <platform.h>
 
-/* 22F3D8 */ void mn_8022F3D8(void);
+#include "dolphin/mtx/types.h"
+
+#include <baselib/object.h>
+
+void mn_8022F3D8(void*, u8, HSD_TypeMask);
+void mn_8022ED6C(HSD_JObj*, Vec3*);
+u32 mn_80229624(u32);            // returns a bitfield of controller inputs
+void mn_80229894(s32, s32, s32); // go to a specific part of the menu?
```

Comment:
If you change it to the following, it will show up in our [docs](https://doldecomp.github.io/melee/mncharsel_8c.html#a75670d84771edc8747d8d06c341c179a) and in `clangd` intellisense this way.
```suggestion
u32 mn_80229624(u32);            ///< @returns a bitfield of controller inputs
void mn_80229894(s32, s32, s32); ///< go to a specific part of the menu?
```

As a style preference, we would rather keep doc comments in the corresponding `*.dox` file (in this case `mn_2295.dox`), which is the same as .c or .h but only contains comments. But keeping a brief comment here for now is of course fine.

Reply:
Also, if you `:source tools/vimrc` in (Neo)vim, you should get Doxygen highlighting.